hack/prom_metric_linter.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2023 Red Hat, Inc.
+#
+#
+set -e
+
+linter_image_tag="v0.0.3"
+
+# Get the metrics list
+metric_families=$(NODE_NAME=node01 go run cmd/handler/main.go --dump-metric-families)
+
+# Run the linter by using the prom-metrics-linter Docker container
+errors=$(podman run -i "quay.io/kubevirt/prom-metrics-linter:$linter_image_tag" \
+    --metric-families="$metric_families" \
+    --operator-name="kubernetes" \
```

Comment:
@qinqon  operator name should be "nmstate"
no need for sub-operator-name or it can also be "nmstate".